src/Resources/Resources/ChangeLog.md

```diff
@@ -19,7 +19,8 @@
 -->
 ## Upcoming Release
 - Remove missing cmdlet referenced in `New-AzResourceGroupDeployment` documentation
-- Updated policy cmdlets to use new api version 2019-01-01
+- Update policy cmdlets to use new api version 2019-01-01
+- Add support for new api version 2019-05-10
```

Comment:
What is the actual, practical impact of thsi change from the customer perspecitve?  What new capabilities are you exposing in cmdlets?

Reply:
ChangeLog updated. Please review.

Reply:
@markcowl Could you please review this PR? I have updated it as requested.